ibis/sql/compiler.py

```diff
@@ -1254,8 +1254,16 @@ def _category_label(expr):
     return stmt.end().name(expr._name)
 
 
+@rewrites(ops.Sum)
```

Comment:
I'm not sure what you're testing, but you may need to look at the ExprTranslator class where the rewrite rules are applied